github/data_source_github_team_repository_test.go

```diff
@@ -0,0 +1,77 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubTeamRepositorty(t *testing.T) {
+
+	randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
+
+	t.Run("Get Repositories By Teams", func(t *testing.T) {
+
+		config := fmt.Sprintf(`
+			resource "github_repository" "repo-test" {
+				name = "tf-acc-repo-%s"
+				auto_init = true
+			
+		  	}
+
+			resource "github_team" "team-test" {
+				name = "tf-acc-test-team01"
+			}
+
+			resource "github_team_repository" "team-repo-test" {
+				repository = "${github_repository.repo-test.id}"
+				team_id = "${github_team.team-test.id}"
+			}
+
+			data "github_team" "example" {
+				slug = "team-test-01"
+			}
+
+   		    output "team_repository_name" {
+				value = data.github_team.example.repositories.0
+			}
+			
+			output "team_repository_numbers" {
+				value = data.github_team.example.repositories.#
+			}
+		`, randomID)
+
+		check := resource.ComposeAggregateTestCheckFunc(
+			resource.TestCheckResourceAttrSet("data.github_team.example", "name"),
```

Comment:
```suggestion
			resource.TestCheckResourceAttr("data.github_team.example", "repositories.#", "1"),
```

A more helpful attribute to check would be the new `repositories.#` field that this PR adds.  When running the test with `TF_LOG=DEBUG`, I'm seeing expected state:

```
data.github_team.example:
  ID = 4838313
  provider = provider.github
  description = 
  name = tf-acc-test-team-zniiu
  node_id = MDQ6VGVhbTQ4MzgzMTM=
  permission = pull
  privacy = secret
  repositories.# = 1
  repositories.0 = tf-acc-repo-zniiu
  slug = tf-acc-test-team-zniiu
```

However, the test fails 😕 .  We should find a way to check this attribute as a next step, but I've run out of time for today.

Reply:
I'll try to look at a solution, but it's possible that I need help!